src/FffGcodeWriter.cpp

```diff
@@ -1592,6 +1542,31 @@ bool FffGcodeWriter::addSupportBottomsToGCode(const SliceDataStorage& storage, L
     return true;
 }
 
+double FffGcodeWriter::supportInterfaceFillAngle(const SliceDataStorage& storage, const EFillMethod pattern, const int layer_number) const
+{
+    if (pattern == EFillMethod::CONCENTRIC)
+    {
+        return 0; //Concentric has no rotation.
+    }
+    if (pattern == EFillMethod::TRIANGLES)
+    {
+        return 90; //Triangular support interface shouldn't alternate every layer.
+    }
+
+    for (const SliceMeshStorage& mesh : storage.meshes)
+    {
+        if (mesh.getSettingInMicrons("support_roof_height") >= 2 * getSettingInMicrons("layer_height"))
```

Comment:
I don't think this logic makes sense for bottom interface angles.

This function is called for both top and bottom interface.